stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/common/internal/LongClickFriendlyLinkMovementMethod.kt

```diff
@@ -39,10 +40,12 @@ internal class LongClickFriendlyLinkMovementMethod private constructor(
     private var isLongClick = false
 
     init {
+        // isXiaomi check fixes issue https://github.com/GetStream/stream-chat-android/issues/3255
```

Comment:
While you're at it, I'd update this to use the /***/ comments, so it's a bit more visible!

Reply:
Updated